Gemfile.lock

```diff
@@ -122,7 +122,7 @@ DEPENDENCIES
   rspec_junit_formatter
 
 RUBY VERSION
-   ruby 3.2.5p208
+   ruby 3.2.6p234
```

Comment:
Did you update this because of the CircleCI Ruby version error? I merged in a PR on Friday that fixes that so you might not need this anymore.

Reply:
Yes. Though since this works, I don't see a compelling reason to roll back. Happy to revert if you think I should

Reply:
Rebased just to make sure nothing breaks and I do have to roll this back after your Friday changes

Reply:
Thanks! Sorry for the extra trouble.